### ISSUE TITLE
Upgrade emoji selection to support latest emojis

### ISSUE THREAD

**Issue by jrainville** [status-im/status-desktop#8979]
### Description

We are missing a lot of common emojis like shrug, upsidedown_smile, etc.

We probably need to upgrade the twemojis version and improve the emoji_json file to include those new files.

**Comment by noeliaSD** [comment]
Moved to next milestone due to team capacity and priorities.

**Comment by caybro** [comment]
@jrainville any pointers and ideas where to start are welcome :) if we still want this update

**Comment by jrainville** [comment]
> @jrainville any pointers and ideas where to start are welcome :) if we still want this update

I tried looking into it a few weeks/months ago and the code has changed a fair bit since we added the emojis, so I was a bit lost.

Also, it seems like Twemoji changed the way they structure their repo.

I think I had created a script that generated the `emojiList.js` file, but maybe it was something that was generated by Twemoji. 
Sadly, it looks like I wasn't very verbose in my PR, so I didn't explain how I got that list https://github.com/status-im/status-desktop/pull/685

At that point, I think if you were to redo it from scratch using the updated Twemoji scripts, it might be simpler than just adding the new emojis and patching our files.
